src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTextToNumber.cs

```diff
@@ -169,6 +169,164 @@ private static (double, ConvertionStatus) ConvertUnsignedNoPercentNumber(string
             return !err ? (0d, ConvertionStatus.InvalidNumber) :
                    IsInvalidDouble(val) ? (0d, ConvertionStatus.InvalidNumber) :
                    (val, ConvertionStatus.Ok);
+        }
+
+        // decimal equivalents
+        // Converts numbers that can be signed and be percentages
+        private static (decimal, ConvertionStatus) ConvertToDecimal(string str, CultureInfo culture)
```

Comment:
>ConvertionStatus [](http://example.com/codeflow?start=33&length=16)

not your change, but shouldn't this be `ConversionStatus`?

Reply:
Yeah, I figured this might be a French thing.  :)  Thought it best to keep the PR to a minimum.  But happy to fix if you'd prefer.

Reply:
yeah not something you need to handle.